content/en/docs/tasks/tls/manual-rotation-of-ca-certificates.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Manual Rotation of CA Certificates
+min-kubernetes-server-version: v1.13
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+This page shows how to manually rotate the certificate authority (CA) certificates.
+
+{{% /capture %}}
+
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+For more information about authentication in Kubernetes, see "[Authenticating](/docs/reference/access-authn-authz/authentication)".
+For more information about best practices for CA certificates, see "[Single root CA](docs/setup/best-practices/certificates/#single-root-ca)".
+
+{{% /capture %}}
+
+{{% capture steps %}}
+## Choose an approach
+There are two ways to rotate or replace CA certificates.
+
+### Replace the cluster (recommended) {#replace-the-cluster}
```

Comment:
I recommend making this a 2nd level heading, so that it's part of the page table of contents.

Reply:
done